turborepo-tests/examples/package.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "turborepo-examples-tests",
+  "scripts": {
+    "test": "./run-example.sh"
+  },
+  "devDependencies": {
```

Doesn't really matter; but since this thing _doesn't work_ without the dependency we should mark it as such.

It's a hard dependency for `turborepo-examples-tests`.

```suggestion
  "dependencies": {
```